docs/user/dashboard/lens.asciidoc

```diff
@@ -54,6 +54,27 @@ For array value fields, the percentage distribution considers each value in the
 
 TIP: *Other* can equal more than 100% by a small amount.
 
+[float]
+[[add-fields-in-lens]]
+===== Add fields
+
+Add and define fields that you want to visualize using the {ref}/modules-scripting-painless.html[Painless scripting language].
+
+. Click *...*, then select *Add field to index pattern*.
++
+[role="screenshot"]
+image:{kib-repo}/discover/images/add-field-to-pattern.png[Dropdown menu located next to index pattern field with item for adding a field to an index pattern, width=50%]
+
+. Enter a *Name* for the field, then select the field *Type*.
+
+. Select *Set value*, then define the field value by emitting a single value using the {ref}/modules-scripting-painless.html[Painless scripting language].
+
+. Click *Save*.
++
+To manage the field, click *...*, select *Manage index pattern fields*, then open the edit options or delete the field.
```

Comment:
Actually we don't want people to use this as the primary way of managing their fields because it requires them to leave Lens. They can edit and delete runtime fields (and change options for other fields) by opening the field information popover and using the edit and delete buttons in there (delete only works for runtime fields):
<img width="779" alt="Screenshot 2021-05-06 at 14 33 35" src="https://user-images.githubusercontent.com/1508364/117298861-21203700-ae78-11eb-9e49-d87bbd4d9e75.png">


The "manage fields" entry in this menu is meant for cases if the user wants to have an overiew over all fields at once.